lxd/storage/backend_lxd.go

```diff
@@ -6230,6 +6242,11 @@ func (b *lxdBackend) BackupCustomVolume(projectName string, volName string, tarW
 		return err
 	}
 
+	// ISO images are read-only, therefore they don't need to be backed up.
+	if volume != nil && drivers.ContentType(volume.ContentType) == drivers.ContentTypeISO {
```

Comment:
Can we make this logic more defensive in general, so define the content types that are supported (possibly using a switch statement) and then if its anything return an error saying the specific content type is not supported for the action.